### PR TITLE
PODCAST PAGE - Display podcast title underneath thumbnail #324

### PR DESCRIPTION
**Pull request by ArindamReact**
I've made the changes according to metion in ticket and made the change to Selectpodcast.vue.

### CODE REVIEW

**Comment by mrjamieeast** [comment]
Please change Font to Bree Serif regular as per the design

**Review by mrjamieeast — APPROVED** [review]

**Comment by mrjamieeast** [comment]
hi @ArindamReact cloudflare build failed

**Review by chrisns — CHANGES_REQUESTED** [review]
add a test

**Comment by ArindamReact** [comment]
> add a test

There is no functionality so no test required, there is only text addition.

**Comment by chrisns** [comment]
> > add a test
> 
> 
> 
> There is no functionality so no test required, there is only text addition.

Assert that the expected text is actually where you expect it to be, proving that it's mapped from the store through to the view correctly?

**Review by chrisns — APPROVED** [review]

**Review by chrisns — APPROVED** [review]

File: cypress/e2e/podcasts.cy.js [hunk 1]
```diff
@@ -270,6 +270,10 @@ describe("newsdesk logged in", () => {
     cy.get("section[slotno=7] textarea").should("not.contain", "new share item")
     cy.get("section[slotno=6] textarea").should("not.contain", "new share item")
   })
+  it("should contain podcast title", () => {
+   cy.visit('http://localhost:3000/')
+   cy.get('#podcast_title').eq(0).should('contain', 'The Smart 7: UK')
+  })
```
Comment (chrisns) [inline cypress/e2e/podcasts.cy.js:276]: Indentation?

**Review by chrisns — CHANGES_REQUESTED** [review]
Not sure why you needed to change the package lock. Also fix indentation

**Comment by ArindamReact** [comment]
> Not sure why you needed to change the package lock. Also fix indentation

because new packages have added in main, and build is failing with current one

**Comment by chrisns** [comment]
> > Not sure why you needed to change the package lock. Also fix indentation
> 
> 
> 
> because new packages have added in main, and build is failing with current one

That's very vague. So you're saying you've reverted updates in main in order to say this is done, without even providing context, perhaps some of those fixes included security fixes.
Please make this change work with the upstream

**Comment by ArindamReact** [comment]
> > > Not sure why you needed to change the package lock. Also fix indentation
> > 
> > 
> > because new packages have added in main, and build is failing with current one
> 
> That's very vague. So you're saying you've reverted updates in main in order to say this is done, without even providing context, perhaps some of those fixes included security fixes. Please make this change work with the upstream

No, I just merge main in my branch because it has conflicts, I've resolved then pushed on PR. Then build fails due to packages are outdated. Then I've updated the package lock using npm install and pushed

**Review by mrjamieeast — APPROVED** [review]

**Review by chrisns — APPROVED** [review]

**Pull request merged**